package.json

```diff
@@ -60,6 +82,7 @@
     "stylelint-config-standard": "^18.0.0",
     "tabbable": "^6.2.0",
     "victory-core": "~36.8.6",
+    "victory-legend": "~36.8.6",
```

Comment:
does this need to be added to the package exclude list?

Reply:
We actually never got an answer in https://github.com/theforeman/foreman/pull/10043/files#r1489694681
So I can copy from the pr and add it? but also not sure if thats the right thing?
Its what Patternfly uses for charts, and in a later pr we're going to use npm overrides to pin them https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides

Reply:
So far, we've been consuming victory via `@theforeman/vendor-core` and thus the pinning here was only for development. Does this change now?